src/store/post.js

```diff
@@ -0,0 +1,16 @@
+export const setPost = post => ({
```

Comment:
```suggestion
export const setPosts = posts => ({
```
я ж не знав що в тебе там їх багато